### ISSUE TITLE
Pagination for filter results

### ISSUE THREAD

**Issue by yamsgithub** [VIDA-NYU/domain_discovery_tool_deprecated#13]

**Comment by britthard** [comment]
@yamsgithub you mentioned before that you needed to do some backend work to get this to work. Is this still the case?

**Comment by yamsgithub** [comment]
@brittainhard Yes...this will require some backend change. But probably a very small one. So please go ahead assuming that you have n number of pages returned and which should be stored on the client side (in javascript) and displayed in m pages at a time. m is specified by the page cap. Does that make sense?
